cores/epoxy/Client.h

```diff
@@ -48,7 +48,7 @@ class Client: public Stream {
         }
 
         #if defined(EPOXY_CORE_ESP8266)
-        const uint8_t* rawIPAddress(IPAddress& addr) const {
+        const uint8_t* rawIPAddress(IPAddress& addr) {
```

Comment:
Please look more closely at the signature for
https://github.com/esp8266/Arduino/blob/master/cores/esp8266/Client.h#L45C25-L45C61

You need to add a 'const' to the argument... otherwise the compiler C++ will complain due to the identical argument signature.